src/jira/client/axios.ts

```diff
@@ -84,16 +100,32 @@ function getErrorMiddleware(logger: Logger) {
 		 * @returns {Promise<Error>} The rejected promise
 		 */
 		(error: AxiosError): Promise<Error> => {
-			if (error?.response) {
-				const status = error.response.status;
+
```

Comment:
Should we feature-flag the change?

Reply:
On the other hand, this is error path, probably doesn't make much sense

Reply:
Yes this. It is error handling path, so should be relatively safe without feature flagging. (In worst case we'll loose some metrics) 

Also was tested in ddev with real failure (404 from deleted Jira instance)